patches/net/minecraft/world/item/component/ItemContainerContents.java.patch

```diff
@@ -0,0 +1,41 @@
+--- a/net/minecraft/world/item/component/ItemContainerContents.java
++++ b/net/minecraft/world/item/component/ItemContainerContents.java
+@@ -146,6 +_,38 @@
+         return this.hashCode;
+     }
+ 
++    // Neo Start
++
++    /**
++     * {@return the number of slots in this container}
++     */
++    public int getSlots() {
++        return this.items.size();
++    }
++
++    /**
++     * Gets a copy of the stack at a particular slot.
++     *
++     * @param slot The slot to check. Must be within [0, {@link #getSlots()}]
++     * @return A copy of the stack in that slot
++     * @throws UnsupportedOperationException if the provided slot index is out-of-bounds.
```

Comment:
Do we want this to throw or just return empty? Either way patching in a helper method to validate the slot index vs just in lining it as we only use it here, feels off (even though yes it is still a single hunk)

Reply:
Since we expose the size as `getSlots` mirroring the behavior of `ItemStackHandler` and throwing an exception is reasonable.